views/js/qtiCreator/widgets/interactions/inlineChoiceInteraction/Widget.js

```diff
@@ -43,14 +45,16 @@ define([
 
     InlineChoiceInteractionWidget.renderInteraction = function(){
 
-        var _this = this,
+        let _this = this,
```

Comment:
Please take a look at OAT [coding guide](https://github.com/oat-sa/taohub-articles/blob/master/articles/Frontend/Architecture/coding-guide.md)  :)
![image](https://user-images.githubusercontent.com/11692751/157850477-91b0aa47-a7c8-4326-bfc5-7bba7e788f83.png)
Also, please check if these variables are going to change their values as the use of `let` suggests.